.travis.yml

```diff
@@ -1,16 +1,27 @@
-language: python
+language: php
 
-python:
-    - "2.7"
+php: 5.6
 
-sudo: false
+sudo: true
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
```

Comment:
the cache cannot be used when you run on the old infrastructure with ``sudo: true`` for public repos